samples/python/server_side.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+import google.oauth2
+
+
+PROJECT_ID = "company-sample-project"
```

Comment:
This is probably a Settle project, right? If so, the name or value should make that a bit clearer.

Reply:
Added a better example for clarification.

This is not necessarily a Settle account. The code is intended to demonstrate using auth from either side, not just from a partner to Settle.